File: RANDRENARIZO tsiry info 04 Projets/MatriceAlgoGausse/src/Ligne.cpp
#include "Ligne.h"
#include <iostream>

Ligne::Ligne()
{
    //ctor
}

Ligne::Ligne(int t)
{
    m_t=t;
    for(int c=0;c<m_t;c++)
       l.push_back(0);//ctor
}
void Ligne::setE(int i,float v)
{
    l[i-1]=v;

}
float Ligne::get(int i)
{
    return l[i-1];
}

void Ligne::afficher()
{
    for(int c=1;c<=m_t;c++)
        cout << Ligne::get(c);

}


Ligne::~Ligne()
{
    //dtor
}

File: RANDRENARIZOTsiryInfo5PrjIIArnaPlolog/RANDRENARIZOTsiryPrjRna/codeSourceC#/prjM2RBF/prjM2RBF/Ligne.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace prjM2RBF
{
    class Ligne
    {


        //private List<double> l =new List<double>();
        private double[] l;
        private int taille;

        public int Taille { get => taille; set => taille = value; }

        public Ligne(int t)
        {
            l = new double[t];
            this.Taille = t;
        }
        public double getE(int indice)
        {
            return l[indice];
        }
        public void setE(int indice,double v)
        {
            l[indice] = v;
        }
        public void afficher()
        {
            for (int c = 0; c < Taille; c++)
                Console.Write(l[c] + "\t");
            Console.WriteLine();
        }

        public void setApartirDUnTableau(double[] tab)
        {
            this.l = tab;
        }

        public double[] getTableau()
        {
            return this.l;
        }




    }
}
